src/components/AbstraxionWallets/index.tsx

```diff
@@ -32,19 +32,27 @@ export const AbstraxionWallets = () => {
     abstractAccount,
     setAbstractAccount,
     setAbstraxionError,
+    config,
   } = useContext(AbstraxionContext) as AbstraxionContextProps;
 
   const { user } = useStytchUser();
   const stytchClient = useStytch();
   const session_jwt = stytchClient.session.getTokens()?.session_jwt;
   const session_token = stytchClient.session.getTokens()?.session_token;
 
+  const API_URL = `${
+    config?.apiUrl || "https://burnt-abstraxion-api.onrender.com"
+  }/api/v1/jwt-accounts/create`;
+  const AUTHENTICATOR = `${
+    config?.projectId || "project-test-185e9a9f-8bab-42f2-a924-953a59e8ff94"
```

Comment:
We need to switch this to the "live" project
```suggestion
    config?.projectId || "public-token-live-59266a62-2720-4452-8984-b65cd54c8838"
```